h/datasetjson.h

```diff
@@ -23,6 +21,9 @@
 
 #define SAF_AUTHORIZATION_READ 0x04
 #define SAF_AUTHORIZATION_UPDATE 0x08
+#define LOG_COMP_ID_MVD_SERVER 0x008F000300010000
```

Comment:
This should not be here. This should be in “logging.h”. It should be something like, “LOG_COMP_ID_DATASETJSON” if it doesnt already exist.